### PR TITLE
Added TCW Percentiles product definition yaml

### PR DESCRIPTION
**Pull request by vnewey**

### CODE REVIEW

File: products/ls_tcw_percentiles/tcw_pct_product.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    name: TM,ETM+,OLI
+
+measurements:
+  # Repeat for all variables
+  - name: TCW_PC_10
+    dtype: float32
+    nodata: -999
+    units: '1' 
+  - name: TCW_PC_50
+    dtype: float32
```
Comment (alexgleith) [inline products/ls_tcw_percentiles/tcw_pct_product.yaml:21]: Is it really float32? That is a big data format for something that's not storing a lot of data!

FC Percentiles use int8: https://explorer.dea.ga.gov.au/product/fc_percentile_albers_annual

**Comment by alexgleith** [comment]
Can you please include the COG product definition too @vnewey 

Not sure how to name them differently...

File: products/ls_tcw_percentiles/tcw_pct_product.yaml (hunk 1 above)
Comment (alexgleith) [reply products/ls_tcw_percentiles/tcw_pct_product.yaml:21]: Ok, it is really float. I'll check out the NetCDF files in a bit.

**Comment by alexgleith** [comment]
Ok sorry, one more change I think, @vnewey:

- Rename the FOLDER to `ga_ls_tcw_percentiles`
- Rename the product to `ga_ls_tcw_percentiles` in both files with a _<format>.yaml at the end.

**Review by omad — APPROVED** [review]

**Pull request merged**